src/components/Image/Image.st.css

```diff
@@ -1,2 +1,32 @@
+:vars {
+  BlurRadius: 3px;
+}
+
+@keyframes blurry-loading {
+  0% {
+    filter: blur(value(BlurRadius));
+  }
+  100% {
+    filter: none;
+  }
+}
+
 .root {
+  -st-states: preload, loaded;
+
+  display: grid;
+}
+
+.root img {
```

Comment:
why don't you use the `MediaImage` style API?

Reply:
I'm not sure there is a style API...

These are the props:
https://github.com/wix/wix-ui/blob/master/packages/wix-ui-core/src/components/media-image/media-image.tsx#L96-L102

I thought about exposing `className` prop - WDYT?

Reply:
Yeah I think a `className` prop is definitely needed no matter the approach.
That way you won't need to target any internal elements.

I would even add it to wix-ui-core before merging this PR.

Reply:
I'll add a `className` for the wrapper div in `Image` & `MediaImage`(since I use both, depends on `src`), but I'll still need to target the native `img` inside.

I can do that in two ways:
1. Expose a prop like `nativeElementClass` on the native `img`
2. Make the native `img` extensible using Stylable

Any preference?